From b9820d8089378db13f08a589bf716a5d747c665c Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Wed, 23 Feb 2022 14:56:42 -0500
Subject: [PATCH] owners.yaml: Update NMI entry with includes

The NMI entry has no includes.  Add files to the entry.

Signed-off-by: Prarit Bhargava <prarit@redhat.com>
---
 info/owners.yaml | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/info/owners.yaml b/info/owners.yaml
index b06921953b2c..0f201155a29e 100644
--- a/info/owners.yaml
+++ b/info/owners.yaml
@@ -4029,6 +4029,13 @@ subsystems:
        email: darcari@redhat.com
        gluser: darcari
    paths:
+       includes:
+          - arch/*/include/asm/nmi.h
+          - arch/x86/kernel/apic/hw_nmi.c
+          - arch/x86/kernel/nmi.c
+          - arch/x86/kernel/nmi_selftest.c
+          - include/linux/nmi.h
+          - lib/nmi_backtrace.c
  - subsystem: NVDIMM SUBSYSTEM
    labels:
      name: nvdimm
-- 
2.35.1

